### PR TITLE
Adds new backpack sprites, replaces the old marine backpack sprites

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Replaces the old marine backpack sprites with new ones made by nixon.
image
Replaces the sprites of all the marine bags.

## Why It's Good For The Game
the bags look better and are more in style.

## Changelog
:cl:
add: Added new backpack sprites
del: Removed old backpack sprites, git will remember them. Nobody else will.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -61. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/game/objects/items/storage/backpack.dm [hunk 1]
```diff
@@ -375,12 +375,17 @@
 
 /obj/item/storage/backpack/marine/satchel
 	name = "\improper TGMC satchel"
-	desc = "A heavy-duty satchel carried by some TGMC soldiers and support personnel."
+	desc = "A heavy-duty satchel carried by some TGMC soldiers and support personnel. This one is in black."
```
Comment (Hughgent) [inline code/game/objects/items/storage/backpack.dm:378]: The descriptions do not need color information, people will see that just by looking at the sprite.

**Review by Hughgent — CHANGES_REQUESTED** [review]

**Review by Hughgent — APPROVED** [review]

**Comment by TiviPlus** [comment]
Merge when the other sprites are done

**Comment by Pariah919** [comment]
@TiviPlus i've added them

**Review by TiviPlus — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-60). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**